src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1892,7 +1892,7 @@ private void AddMultipartContent(object fieldName, object fieldValue, MultipartF
             // Treat Strings and other single values as a StringContent.
             // If enumeration is false, also treat IEnumerables as StringContents.
             // String implements IEnumerable so the explicit check is required.
-            if (enumerate == false || fieldValue is string || !(fieldValue is IEnumerable))
+            if (enumerate == false || fieldValue is string || fieldValue is not IEnumerable)
```

Comment:
@iSazonov @daxian-dbw are we better off using `LanguagePrimitives.IsEnumerable()` here, since that already has special casing for strings and other "not PS-enumerable" types built in?

Reply:
Looking how AddMultipartContent() is used it would be not a fix in only the line. So we need to consider a specific scenario to get full understanding. If you know such scenario please open new issue.